groups/ntc/ntcu/ntcu_timestampcorrelator.h

```diff
@@ -0,0 +1,160 @@
+// Copyright 2023 Bloomberg Finance L.P.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef INCLUDED_NTCU_TIMESTAMPCORRELATOR
+#define INCLUDED_NTCU_TIMESTAMPCORRELATOR
+
+#include <bsls_ident.h>
+BSLS_IDENT("$Id: $")
+
+#include <ntccfg_inline.h>
+#include <ntccfg_platform.h>
+#include <ntsa_timestamp.h>
+#include <bdlb_nullablevalue.h>
+#include <bsls_timeinterval.h>
+#include <bslstl_pair.h>
+#include <bsl_cstdint.h>
+
+namespace BloombergLP {
+namespace ntcu {
+
+/// @internal @brief
```

Comment:
Include a period at the end of each phrase; the documentation style uses reasonably-phrased sentences.

Reply:
Done